components/formats-gpl/src/loci/formats/in/CellH5Reader.java

```diff
@@ -586,11 +589,11 @@ private void parseROIs(int s) {
           store.setRectangleHeight((double) height, roiManagerRoiIndex, 0);
 
           store.setRectangleTheT(
-            new NonNegativeInteger(roiTime), roiManagerRoiIndex, 0);
+            new NonNegativeInteger(roiTime + 1), roiManagerRoiIndex, 0);
           store.setRectangleTheC(
-            new NonNegativeInteger(roiChannel), roiManagerRoiIndex, 0);
+            new NonNegativeInteger(roiChannel + 1), roiManagerRoiIndex, 0);
           store.setRectangleTheZ(
-            new NonNegativeInteger(roiZSlice), roiManagerRoiIndex, 0);
+            new NonNegativeInteger(roiZSlice + 1), roiManagerRoiIndex, 0);
```

Comment:
0-based indexes are correct here (see e.g. http://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2015-01/ROI_xsd.html#Shape_TheZ).  If ImageJ is expecting the index to start from 1, then this should be fixed in ROIHandler instead.